specification/keyvault/Security.KeyVault.Secrets/client.tsp

```diff
@@ -34,3 +34,26 @@ using KeyVault;
 
 @@clientName(SecretRestoreParameters.secretBundleBackup, "SecretBackup", "go");
 @@clientName(SecretBundle.kid, "KID", "go");
+
+// Rust configuration
+@@clientName(SecretProperties, "OmitSecretProperties", "rust");
+@@clientName(SecretItem, "SecretProperties", "rust");
+
+@@clientName(DeletedSecretBundle, "DeletedSecret", "rust");
+@@clientName(DeletedSecretItem, "DeletedSecretProperties", "rust");
+@@clientName(SecretBundle, "Secret", "rust");
+@@clientName(SecretRestoreParameters, "RestoreSecretParameters", "rust");
+@@clientName(SecretSetParameters, "SetSecretParameters", "rust");
+@@clientName(SecretUpdateParameters, "UpdateSecretPropertiesParameters", "rust");
+@@clientName(DeletedSecretListResult,
+  "DeletedSecretPropertiesListResult",
+  "rust"
+);
+@@clientName(SecretListResult, "ListSecretPropertiesResult", "rust");
+
+@@clientName(getDeletedSecrets, "ListDeletedSecretProperties", "rust");
+@@clientName(getSecrets, "ListSecretProperties", "rust");
+@@clientName(getSecretVersions, "ListSecretPropertiesVersions", "rust");
+@@clientName(updateSecret, "UpdateSecretProperties", "rust");
```

Comment:
I know these are matching other languages, but I dislike the "Properties" postfix.

Reply:
Makes sense